src/DOMupdates.js

```diff
@@ -53,11 +43,11 @@ let domUpdates = {
     if (answer === true) {
       $("#js-show-answer").text('YOU GUESSED CORRECTLY!');
       $(`#${id}`).html('')
-      console.log(id)
       $('#js-guess-input').val('')
```

Comment:
Since this line is repeated in both conditions, you can bring it outside of the if/else to reduce the repetition.